vnext/ReactUWP/Views/TouchEventHandler.cpp

```diff
@@ -74,13 +76,13 @@ void TouchEventHandler::OnPointerPressed(
 
   // Only if the view has a Tag can we process this
   int64_t tag;
-  winrt::FrameworkElement sourceElement(nullptr);
+  winrt::UIElement sourceElement(nullptr);
```

Comment:
>UIElement [](start = 9, length = 9)

why are we changing all FE to UIE? / what property are you missing in FE?

Reply:
Actually the opposite -- FE inherits from UIE. Support for input is part of UIE, so this is a generalization. It ensures that in the rare case that an app has a Xaml component in its visual tree that inherits directly from UIE instead of through FE, we are still able to walk upwards from that element to find a React component and fire events appropriately.

With the previous code, if a user moused over such an element and it was the OriginalSource for the input events, we would bail out and never fire any events for it or its ancestors.

Is it a false assumption that there could ever be such an element? 

---
In reply to: [408675041](https://github.com/microsoft/react-native-windows/pull/4574#discussion_r408675041) [](ancestors = 408675041)

Reply:
Technically this change wasn't necessary as there are no classes that inherit directly from UIElement.  However I don't see any harm in doing it.

---
In reply to: [409072995](https://github.com/microsoft/react-native-windows/pull/4574#discussion_r409072995) [](ancestors = 409072995,408675041)